python/_fio_plugin/requirements.txt

```diff
@@ -0,0 +1,2 @@
+arcaflow-plugin-sdk @ git+https://github.com/arcalot/arcaflow-plugin-sdk-python.git@main ; python_version >= "3.9"
```

Comment:
The SDK will install directly from PyPi. You shouldn't need to point to the github URL, and beyond that we definitely should point to a release and not the main branch.